LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright Contributors to the OpenColorIO Project.
+Copyright Contributors to the OpenColorIO-Configs-ACES Project.
```

Comment:
We should maybe discuss this further at the next meeting.

Reply:
Yeah no worries, there so that there is something and we avoid being in the previous situation ;)